<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblStartTime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="teStartTime">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>118</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblEndTime">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>끝</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="teEndTime">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>118</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPeriod">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>정기알람
 주기(분)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbPeriod">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStatus">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwAlarms">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPeriod_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>수시 알람
 최대주기(초)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbFrequentPeriod">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
